src/trustymail/cli.py

```diff
@@ -61,20 +62,16 @@
 from . import trustymail
 from ._version import __version__
 
-# The default ports to be checked to see if an SMTP server is listening.
-_DEFAULT_SMTP_PORTS = {25, 465, 587}
-
 
 def main():
     """Perform a trustymail scan using the provided options."""
     args = docopt.docopt(__doc__, version=__version__)
+    print('args as interpreted by trustymail:', args)
+
+    # Write the arguments to a file for use by the trustymail library
+    with open('env.json', 'w') as env:
+        json.dump(args, env)
```

Comment:
This is worse than a monkey patch and a complete non-starter to me as it breaks the API of this package.

Reply:
Noted; I'll make an adjustment.

Reply:
@mcdonnnj Thanks for the feedback. What are your thoughts on adding arguments (readonly, filename) to domain.get_psl  and moving the call to get_psl to cli.py?